CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [3.0.0-beta1] - 2023-07-13
+
+This is the first public beta release of our new `ingress-nginx` chart.
+
+Even though we put a lot of effort into testing every change, we do not recommend you to install this release in a production environment.
+
+If you are currently using `v2.x.x`, this release includes breaking changes from [`v3.0.0-alpha1`](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-alpha1). We set up a [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0-beta1/migration.md) to make the upgrade as smooth as possible.
+
+Every kind of feedback regarding this release, its changes or our [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0-beta1/migration.md) is very welcome!
```

Comment:
```suggestion
Every kind of feedback regarding this release, its changes, or our [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0-beta1/migration.md) is very welcome!
```

Reply:
Why? :D I never got this...

Reply:
https://prowritingaid.com/Comma#:~:text=Listing%20commas%20can%20separate%20lists,a%20list%20of%20dependent%20clauses.

Well, Oxford it is then...